src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdater.java

```diff
@@ -61,15 +66,16 @@ public SuccessfulUpdateResponse update(CaseUpdateRequest caseUpdateRequest) {
             originalCase.bulkScanCaseReference
         );
 
+        final List<String> warnings = updatedCaseValidator.getWarnings(newCase);
```

Comment:
Can you describe the plan of how this will be used to return errors instead of warnings?
I don't see where this is going.

Reply:
it will look like this:
`        if (caseUpdateRequest.isAutomatedProcess && !warnings.isEmpty()) {
            throw HttpClientErrorException.create(
                HttpStatus.UNPROCESSABLE_ENTITY,
                "unprocessable entity message",
                null,
                String.join(",", warnings).getBytes(),
                null
            );
        } else {
            return new SuccessfulUpdateResponse(
                new CaseUpdateDetails(
                    // This is just a sample implementation.
                    // You can use different event IDs based on the changes made to a case...
                    EVENT_ID,
                    newCase
                ),
                warnings
            );
        }
`

Reply:
Thanks.
Just remember such an error would not follow the api spec.
You can just throw `InvalidExceptionRecordException` I think